index.html

```diff
@@ -12220,14 +12228,47 @@ <h3>Examples</h3>
 
 <section id="security" class="appendix">
   <h3>Security Considerations</h3>
-  <p class="ednote">Consider requirements from <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review Questionnaire: Security and Privacy</a>.</p>
-  <p>See, <a href="#iana-considerations" class="sectionRef"></a></p>
+  <p>See, <a href="#iana-security">Security Considerations</a> in <a href="#iana-considerations" class="sectionRef"></a>.</p>
```

Comment:
We did say on the call that adding some paragraph on the future need for context as well as JSON data file signature might be a good idea. Should it be here?

Reply:
Do we want to add a reference to issue #86 ?